protocol-dashboard/src/containers/Home/Home.tsx

```diff
@@ -1,108 +1,226 @@
+import { RewardsTimingCard } from 'components/RewardsTimingCard/RewardsTimingCard'
+import { StatLabel } from 'components/StatLabel/StatLabel'
+import { createStyles } from 'utils/mobile'
 import desktopStyles from './Home.module.css'
 import mobileStyles from './HomeMobile.module.css'
-import { createStyles } from 'utils/mobile'
-import { useIsMobile } from 'utils/hooks'
+import { InfoCard } from './InfoCard'
 
 const styles = createStyles({ desktopStyles, mobileStyles })
 
 const messages = {
   title: 'Overview',
+  uniqueMonthlyUsers: 'Unique Monthly Users',
+  globalStakedAudio: `Global Staked ${TICKER}`,
+  estimatedRewardRate: `Estimated ${TICKER} Reward Rate`,
+  apiCalls: 'Monthly API Calls',
   recentProposals: 'Recent Proposals',
   noProposals: 'No Recent Proposals',
   viewAllProposals: 'View All Proposals',
-  wtfIsAudius: 'WTF is Audius?',
-  wtf1: `Audius is a digital streaming service that connects fans directly with artists and exclusive new music`,
-  wtf2: `It does this by being fully decentralized: Audius is owned and run by a vibrant, open-source community of artists, fans, and developers all around the world. Audius gives artists the power to share never-before-heard music and monetize streams directly. Developers can build their own apps on top of Audius, giving them access to one of the most unique audio catalogs in existence.`,
-  wtf3: `Backed by an all-star team of investors, Audius was founded in 2018 and serves millions of users every month, making it the largest non-financial crypto application ever built.`
+  wtfIsAudius: 'What is Audius?',
+  wtf1: `Audius is not just a digital streaming platform; it's a revolution that connects fans and artists, bringing exclusive new music to your fingertips.`,
+  launchAudius: 'Launch Audius',
+  trustedNameTitle: 'Trusted Name in Music Innovation',
+  trustedNameDescription:
+    'Launched in 2018 and backed by an all-star team of investors, Audius has grown to serve millions of users monthly, standing as the largest non-financial crypto application to date.',
+  trustedNameCTA: 'Open Audio Foundation',
+  powerOfDecentralizationTitle: 'The Power of Decentralization',
+  powerOfDecentralizationDescription:
+    'Owned and operated by a diverse community of artists, fans, and developers globally, Audius empowers artists to share unique content and monetize streams directly.',
+  powerOfDecentralizationCTA: 'Read The White Paper',
+  developerFriendlyTitle: 'Developer Friendly',
+  developerFriendlyDescription:
+    'Unlock the potential of music innovation with Audius. Our robust API gives access to hundreds of thousands of songs, offering unparalleled opportunities for creativity.',
+  developerFriendlyCTA: 'API Documentation'
 }
 
 const Home = () => {
   const { isLoggedIn } = useAccount()
   const { recentProposals } = useProposals()
   const pushRoute = usePushRoute()
-  const isMobile = useIsMobile()
+  const { typography } = useTheme() as HarmonyTheme
```

Comment:
is this typecast necessary?

Reply:
yeah for some reason the type is wrong when imported from harmony. will add a comment